### PR TITLE
Updated ygot and goyang version to the new release for each model

### PR DESCRIPTION
**Pull request by frouzbeh**

### CODE REVIEW

**Review by SeanCondon — COMMENTED** [review]
I was expecting to see a change in `templates/go.mod.tpl` also - that should drive the change in these files, and I would have expected to see the `generated.go` files change too.

Also in the `build/model-compiler/Dockerfile` there is a ref to the generator tool that needs to be changed

File: templates/go.mod.tpl [hunk 1]
```diff
@@ -10,8 +10,8 @@ require (
 	github.com/onosproject/onos-api/go v0.9.14
 	github.com/onosproject/onos-lib-go v0.8.13
 	github.com/openconfig/gnmi v0.0.0-20210914185457-51254b657b7d
```
Comment (SeanCondon) [inline templates/go.mod.tpl:12]: I think you could change this to `v0.0.0-20220617175856-41246b1b3507` also. The compiler corrects it when it does `go mod tidy` but it's better to keep it current here too

**Review by SeanCondon — APPROVED** [review]
thanks for the update - just 1 minor comment

File: templates/go.mod.tpl (hunk 1 above)
Comment (frouzbeh) [reply templates/go.mod.tpl:12]: Sorry I didn't notice this comment, I'll fix it in the next patch.

**Pull request merged**